### DISCUSSION TITLE
Ever be an extension remover that would work without inspect or anything

### DISCUSSION THREAD

**Discussion by AdanGonz2444** [3kh0/ext-remover#452] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Will there ever be an extension remover that can remove any extension you want that doesn't require inspect cause my school took off things like view source code and inspect

**Comment by piplup7575** [comment, marked as answer]
What version are you on? You can use the dns method of ltbeef if you are under version 106 and on home wifi

**Reply by Darcii** [reply]
anything for ver 108.0.5359.111 ive been trying to keep my chromebook un-updated in hope of a new exploit being shared.

**Reply by Aka-but-nice** [reply]
What blocker do you have? If you have either Securly, Blocksi, Cisco Umbrella, or iBoss, Try New Point Blank, which is listed on the github main page.

**Reply by AdanGonz2444** [reply]
My school doesn't use blockers the wifi blocks stuff there is one extension which is ck but that is rare

> What blocker do you have? If you have either Securly, Blocksi, Cisco Umbrella, or iBoss, Try New Point Blank, which is listed on the github main page.

**Reply by AdanGonz2444** [reply]
> What version are you on? You can use the dns method of ltbeef if you are under version 106 and on home wifi

Im above 106 but I made it so my chromebook wont update so if there is a exploit to disable extensions

**Reply by Darcii** [reply]
> What blocker do you have? If you have either Securly, Blocksi, Cisco Umbrella, or iBoss, Try New Point Blank, which is listed on the github main page.

Unlucky me I have goguardian as the first filter, and cisco unbrella as the second filter. Its weird I dont see cisco as a extension i think its ON the school wifi.
